units/meson.build

```diff
@@ -262,7 +262,8 @@ foreach tuple : in_units
 
         # we do this here because install_data does not accept custom_target output
         conds = tuple[1].split(' ')
-        install = ((conds.get(0, '') == '' or conf.get(conds[0]) == 1) and
+        install = (conds.get(0, '') == 'not' ? conf.get(conds[1]) != 1 :
+                   (conds.get(0, '') == '' or conf.get(conds[0]) == 1) and
```

Comment:
I don't think this makes sense. The configuration condition is a boolean, and adding a second way to specify this boolean doesn't add any new functionality.

Reply:
There is no negated condition possible?